qos-service-impl/src/test/java/com/palantir/atlasdb/qos/ratelimit/QosRateLimiterTest.java

```diff
@@ -32,7 +32,8 @@
     private static final long START_TIME_MICROS = 0L;
     private static final long MAX_BACKOFF_TIME_MILLIS = 10_000;
 
-    RateLimiter.SleepingStopwatch stopwatch = mock(RateLimiter.SleepingStopwatch.class);
+    com.palantir.atlasdb.qos.ratelimit.guava.RateLimiter.SleepingStopwatch stopwatch = mock(
```

Comment:
do we need the full paths here?  can we import them?

Reply:
oops, this was unintentional